pages/blog/css-with-react.mdx

```diff
@@ -0,0 +1,274 @@
+---
+title: 'How Should I Style My React Application?'
+publishedAt: '2020-09-15'
+summary: 'TODO'
+image: '/static/images/css-with-react/banner.png'
+---
+
+The React ecosystem is unopinionated. **This is both it's greatest strength and weakness.**
+
+Over the years, I tried about seven different ways to style my React applications. Each solution has their own pros and cons.
+Most comparions between CSS / CSS-in-JS and their ecosystem fail to consider the scale of the application and the size of the team.
+
+Let's go on a journey and learn how CSS has evolved to bring us to its current state. I'll share my journey and then discuss pros / cons of
+different approaches for styling React applications after. I'll try to make this comparison as unopinionated as possible.
+
+## Vanilla CSS
+
+Like most, I started web development with vanilla CSS. You have to learn the basics first. All we need is an HTML and CSS file.
+
+```html:index.html
+<h1 class="header">Welcome</h1>
+```
+
+```css:index.css
+.header {
+  font-size: 32px;
+}
+```
+
+As I built more applications, I began to understand some of the drawbacks.
+
+- **Problem.** todo
+- **Specificity.** Since CSS has a global namespace, you can unintentionally target too many elements.
```

Comment:
```suggestion
- **Global Namespace.** Since CSS has a global namespace, you can unintentionally target too many elements.
```

Probably global namespace is more accurate. Specificity is the weight of a selector